src/Areas/Workbooks/Commands/Workbook/ListWorkbooksCommand.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.Workbooks.Models;
+using AzureMcp.Areas.Workbooks.Options.Workbook;
+using AzureMcp.Areas.Workbooks.Services;
+using AzureMcp.Commands.Subscription;
+using AzureMcp.Commands;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Workbooks.Commands.Workbook;
+
+public sealed class ListWorkbooksCommand(ILogger<ListWorkbooksCommand> logger) : SubscriptionCommand<ListWorkbooksOptions>
+{
+    private const string CommandTitle = "List Workbooks";
+    private readonly ILogger<ListWorkbooksCommand> _logger = logger;
+
+    public override string Name => "list";
+
+    public override string Description =>
+        """
+        List all workbooks in a specific resource group. This command retrieves all workbooks available
+        in the specified resource group within the given subscription. Resource group is required.
+        """;
+
+    public override string Title => CommandTitle;
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(_resourceGroupOption);
+    }
+
+    protected override ListWorkbooksOptions BindOptions(ParseResult parseResult)
+    {
+        var options = base.BindOptions(parseResult);
+        options.ResourceGroup = parseResult.GetValueForOption(_resourceGroupOption);
+        return options;
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = CommandTitle)]
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var options = BindOptions(parseResult);
+
+        try
+        {
+            if (!Validate(parseResult.CommandResult, context.Response).IsValid)
+            {
+                return context.Response;
+            }
+
+            var workbooksService = context.GetService<IWorkbooksService>();
+            var workbooks = await workbooksService.ListWorkbooks(options.Subscription!, options.ResourceGroup!, options.RetryPolicy);
```

Comment:
haven't dug through yet, but is this just using ARM apis for list?  if so, those are SUPER slow compared to ARG.  this should be using ARG for list if at all possible.

(querying across a whole sub/etc to list all workbooks makes ARM fan out and query each region for any workbooks in that region, wait for them to all return, and union the results together)